src/pudl/constants.py

```diff
@@ -2706,8 +2707,11 @@
         'primary_purpose_naics_id': pd.Int64Dtype(),
         'prime_mover_code': pd.StringDtype(),
         'real_time_pricing_program': pd.BooleanDtype(),  # Added by AES for DP table
+        'rec_revenues': float,  # Added by AES for GP table
+        'rec_sales_mwh': float,  # Added by AES for GP table
         'regulatory_status_code': pd.StringDtype(),
         'report_date': 'datetime64[ns]',
+        'revenues': float,  # Added by AES for GP table
```

Comment:
This seems like it's probably too generic of a column name. Might need to change the `xlsx_map` info -- too easy to end up with a collision with other kinds of revenue. Not sure what the right name is though -- but we need these column names to be unique/consistent across all of the EIA 860, 861, and 923 data (and maybe other EIA data too someday?). I think we've generally tried to stay away from using plurals too.